benches/cub/device/merge_sort/custom_less_thrust_comparison.cu

```diff
@@ -0,0 +1,85 @@
+#include <nvbench/nvbench.cuh>
+
+#include <thrust/device_vector.h>
+#include <thrust/sequence.h>
+#include <thrust/random.h>
+#include <thrust/shuffle.h>
+
+#include <thrust/detail/temporary_array.h>
+
+#include <cub/device/device_merge_sort.cuh>
+
+/*
+ * Arithmetic types with the custom comparator lead to merge sort usage.
```

Comment:
I'm not sure what this means -- what is this benchmark for?

It looks like this is a temporary benchmark for testing the ported code, and wouldn't need to be included in the regression benchmark suite.